### PR TITLE
New term Vertebral Artery Dissection

### LINKED ISSUE #7593: Request for new term [vertebral artery dissection]
### Label

vertebral artery dissection

### Synonyms

_No response_

### Synonym type

None

### Definition

Vertebral artery dissection (VAD) is a rare cause of stroke in the general population; however, represents one of the more common causes of stroke in patients younger than 45 years of age. Its signs and symptoms can be vague, and diagnosis can be elusive. Spontaneous dissections have been reported. However, incidental minor trauma often precipitates this potentially dangerous condition. Often some neck distortion such as chiropractic manipulation, bending of the neck, or blunt trauma causes the dissection. The dissection of the artery may ultimately lead to a stroke which often can be delayed for days following the acute dissection.  Source: https://www.ncbi.nlm.nih.gov/books/NBK441827/

### Parent term

cervical artery dissection

### Children term(s)

_No response_

### ORCID Identifier

_No response_

### Website URL

https://www.ncbi.nlm.nih.gov/books/NBK441827/  https://www.cureus.com/articles/51091-vertebral-artery-dissection-a-pain-in-the-neck#!/ https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2588305/   https://emedicine.medscape.com/article/761451-overview

### Additional comments

We received a message from an inquirer who is a stroke survivor asking if we  had this condition listed at NORD.  Currently it is not listed in NORD and it is not in Orphanet either (I will write a ticket to them for their consideration). Orphanet does list familial cervical artery dissection  (note that cervical artery dissection occurs when there's a tear in either a carotid or vertebral artery).  It is a rare condition.
Please let me know your thoughts.  Thank you, Gioconda Alyea

### PR DESCRIPTION
**Pull request by yshwetar**
Closes #7593

Following in new term

- Definition with 1 PMID
- 2 PMID dbxref
- Child term of cervical artery dissection

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -574661,6 +574661,16 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7379" xsd:anyURI
 
+[Term]
+id: MONDO:1040001
+name: vertebral artery dissection
+def: "Vertebral artery dissection (VAD) is a rare cause of stroke in the general population; however, represents one of the more common causes of stroke in patients younger than 45 years of age. Its signs and symptoms can be vague, and diagnosis can be elusive. Spontaneous dissections have been reported. However, incidental minor trauma often precipitates this potentially dangerous condition. Often some neck distortion such as chiropractic manipulation, bending of the neck, or blunt trauma causes the dissection. The dissection of the artery may ultimately lead to a stroke which often can be delayed for days following the acute dissection." [PMID:28722857]
+xref: PMID:19096659
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:574668]: xref should refer to an ID in another terminology (e.g. OMIM:##, Orphanet:###). Not PMID. 
PMID can be used as an annotation to a definition or synonym

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -574661,6 +574661,16 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
...
+xref: PMID:33659122
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:574669]: xref should refer to an ID in another terminology (e.g. OMIM:##, Orphanet:###). Not PMID. 
PMID can be used as an annotation to a definition or synonym

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -574661,6 +574661,16 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
...
+is_a: MONDO:0006061 ! cervical artery dissection
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:574670]: source annotation is missing

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@yshwetar I added some comments to bring your attention to some things to keep in mind. Mostly related to x-refs and source missing for the SubclassOf.

That being said, I don't think this term being requested should be added into Mondo because it is not a disease. As the definition states, it is a "rare cause of stroke", and is probably more of a condition or an event. See more information about exclusion reasons [here](https://mondo.readthedocs.io/en/latest/editors-guide/exclusion-reasons/)
What do you think?

**Comment by yshwetar** [comment]
Closing this branch as we concluded VAD is not a disease, and an external factor, #7593

**Pull request closed without merging**